evennia/accounts/accounts.py

```diff
@@ -189,6 +189,10 @@ def nicks(self):
     @lazy_property
     def sessions(self):
         return AccountSessionHandler(self)
+        
+    @lazy_property
+    def characters(self):
+        return self.db._playable_characters
```

Comment:
Django templates cannot access attributes with underscores; this is a valuable field anyway so I'm not sure why it's private/hidden.

Reply:
Hmm. The locks thing makes sense when determining who can puppet what, but it doesn't lend itself well to identifying which players "own" which characters at the db level. I think the reason myself and others have just taken advantage of this attribute is because it's simple and intuitive. Some sort of wrapper would be beneficial. Not sure how to proceed with this though.